test/extended/operators/certs.go

```diff
@@ -49,6 +56,82 @@ var _ = g.Describe("[sig-arch][Late]", func() {
 		o.Expect(err).NotTo(o.HaveOccurred())
 	})
 
+	g.It("all tls artifacts must be known", func() {
```

Comment:
I see three tests here which should be separated with `By`s.

1. registered/known
2. owned
3. described